ocaml/sdk-gen/csharp/FriendlyErrorNames.resx

```diff
@@ -354,16 +354,6 @@
   <data name="RBAC_PERMISSION_DENIED" xml:space="preserve">
     <value>Your current role is not authorized to perform this action.
 Action: {0}</value>
-  </data>

```

Comment:
These here are part of special error handling in XenCenter and don't belong in the SDK.